src/main/plugins/currency-converter-plugin/currency-converter-plugin.ts

```diff
@@ -25,12 +25,13 @@ export class CurrencyConverterPlugin implements ExecutionPlugin {
     }
 
     public isValidUserInput(userInput: string, fallback?: boolean | undefined): boolean {
+        const keywords = ["in", "to"];
```

Comment:
I decided to declare the keywords up here, instead of inline, am happy to change if you think `["in", to"].includes(...)` would read better?